plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -328,11 +328,14 @@ private List<DeltaLakeSplit> splitsForFile(
 
     private static Location buildSplitPath(Location tableLocation, AddFileEntry addAction)
     {
-        // paths are relative to the table location and are RFC 2396 URIs
+        // paths are relative to the table location or absolute in case of shallow cloned table and are RFC 2396 URIs
         // https://github.com/delta-io/delta/blob/master/PROTOCOL.md#add-file-and-remove-file
         URI uri = URI.create(addAction.getPath());
```

Comment:
Could you add javadoc to `AddFileEntry.getPath` method that explains the above behavior? 

Also, it would be nice to leave a comment in `TableChangesSplitSource` & `VacuumProcedure`. Those usages expect relative path.

Reply:
please let me know if the comment suffice, or any more details are required to be added